reports/docusign-autopopulation/index.md

```diff
@@ -0,0 +1,134 @@
+3.	Recipient tab
+	- tabLabel - Data Label defined in DocuSign
+	- value - the value you want to insert (use merge syntax here!)
+	- font (optional) - choose a font to stylize your text. (Supported Fonts: Arial, Arial, ArialNarrow, Calibri, CourierNew, Garamond, Georgia, Helvetica,   LucidaConsole, Tahoma, TimesNewRoman, Trebuchet, Verdana, MSGothic, MSMincho, Default)
+
+#### Example Request Body:  
+ 
+    {
+        "status": "sent",
+        "emailSubject": "Please DocuSign this Form",
+        "templateId": "<Your template Id>",
+        "templateRoles": [{
+            "name": "{{RecipientName}}",
+            "email":"recipient@gmail.com",
+            "roleName": "Recipeint1",
+            "recipientId": "1",
+            "routingOrder": "1",
+            "tabs": {
+                "textTabs": [{
+                    "tabLabel": "EmployeeName",
+                    "value": "{{employeeName}}",
+                    "Font": "Calibri"},
+                    {
+                    "tabLabel": "EffectiveDate",
+                    "value": "{{Date}}",
+                    "Font": "Calibri"},                        
+                    {
+                    "tabLabel": "Notes",
+                    "value": "{{Notes}}",
+                    "Font": "Calibri"
+                    }]
+            }
+            
+        },
+        {
+            "name": "{{employeeName}}",
+            "email": "{{employeeEmail}}",
+            "roleName": "Employee",
+            "recipientId": "2",
+            "routingOrder": "2"
+        }]
+    }
+
+### See it in action!: 
+1.	In your Skuid page:
+2.	Remove any existing rows in the REST model and create a new row. 
+3.	Create an action that saves the new row. 
+4.	Look for an email from DocuSign (if you are the first recipient) 
+5.	You did it!
+
+### Promote your key to a production account: 
+In order to send real DocuSign documents, you must promote your API key to a production account. Once it has been promoted, you will need to change some of the data source settings to match the new URL etc. 
+To do this:
+1.	[Promote your key](https://support.docusign.com/guides/ndse-admin-guide-api-and-keys) 
+2.	With the new key in the production account, follow steps 2-5 under the DocuSign section above. 
+3.	You will also need to transfer your DocuSign template, which is made simple by selecting the download option on the template you want to transfer and then selecting new > upload template in the production account. 
+4.	In Skuid configure, you will need to modify the authentication provider and data source
+	- New Data source URL: `https://na2.docusign.net/restapi/v2` (may change based on your instance)
+	- New Authorize endpoint: `https://account.docusign.com/oauth/auth`
+	- New Token endpoint: `https://account.docusign.com/oauth/token`
+	- Also make sure to update the Client Id and Client secret if they have changed.
+6.	Test it out! 
```

Comment:
So, fun Markdown fact: It doesn't pay attention to the number you've placed, instead rendering whatever the next number is.

So while having the `6.` here is incorrect, it still renders correctly. But for consistency's sake, we should probably fix it:

```suggestion
5.	Test it out! 
```